website/pages/en/querying/graphql-api.mdx

```diff
@@ -106,7 +118,7 @@ Query 10 `Token` entities, offset by 10 places from the beginning of the collect
 
 #### Example using `first` and `id_ge`
 
-If a client needs to retrieve a large number of entities, it is much more performant to base queries on an attribute and filter by that attribute. For example, a client would retrieve a large number of tokens using this query:
+If a client needs to retrieve a large number of entities, it's best to base queries on an attribute and filter by that attribute. For example, a client could retrieve a large number of tokens using this query:
```

Comment:
I feel like we lose some information here, why is it best? Because it's more performant.